EventAggregator.Net.SampleUsage/EventAggregator.Net.SampleUsage.csproj

```diff
@@ -34,8 +36,33 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.Threading.Tasks, Version=1.0.12.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">

```

Comment:
This project targets .NET Client Profile framework that has been discontinued (see [HERE](https://msdn.microsoft.com/en-us/library/cc656912(v=vs.110).aspx))